package-lock.json

```diff
@@ -3104,9 +3104,9 @@
       "dev": true
     },
     "node-abi": {
-      "version": "2.19.1",
-      "resolved": "https://registry.npmjs.org/node-abi/-/node-abi-2.19.1.tgz",
-      "integrity": "sha512-HbtmIuByq44yhAzK7b9j/FelKlHYISKQn0mtvcBrU5QBkhoCMp5bu8Hv5AI34DcKfOAcJBcOEMwLlwO62FFu9A==",
+      "version": "2.18.0",
```

Comment:
Where is the change to `package.json`? We should pin the version there, not just the lock file

Reply:
We don't depend on it directly, it's a dependency of `prebuild`.

Reply:
is it not possible to explicitly pin it in our package.json even if its a transient dependency? I'm concerned that the next time someone runs `npm install` it will "fix" this and revert your change

Reply:
It's certainly possible to add it to our `package.json` but I don't think it's necessarily the correct solution since it's not a direct dependency. Regarding your concern, subsequent `npm installs` should pull the version from `package-lock.json`, someone would have to delete the current `package-lock.json` before running an `npm install`, or explicitly run `npm install prebuild@latest`, in order to update the version in the lock file.